manifest/reader.go

```diff
@@ -415,6 +422,8 @@ func prefixModules(mods []*pbsubstreams.Module, prefix string) {
 				input.Store.ModuleName = prefix + PrefixSeparator + input.Store.ModuleName
 			case *pbsubstreams.Module_Input_Map_:
 				input.Map.ModuleName = prefix + PrefixSeparator + input.Map.ModuleName
+			case *pbsubstreams.Module_Input_Params_:
+				input.Params.ModuleName = prefix + PrefixSeparator + input.Params.ModuleName
```

Comment:
Params doesn't need a `ModuleName`. So in the `modules.proto` file:
```
+    message Params {
+      string module_name = 1;
+      string value = 2;
+    }
```
should only be:
```
+    message Params {
+      string value = 1;
+    }
```

The other ones like map and store, are _pointers_ to those modules.. so that's why we need the name. Params are not pointers to any  module, they just are specified or not.